suricata/update/main.py

```diff
@@ -381,6 +386,9 @@ def fetch(self, url):
                     "will use latest cached version: %s", url, err)
                 return self.extract_files(tmp_filename)
             raise err
+        except IOError as err:
+            logger.error("Failed to copy file %s", err)
+            sys.exit(1)
```

Comment:
So, this block would have sufficed. However,
`err` will never have the filename. So, the error output would look absurd. Something like,
```
21/5/2019 -- 18:19:05 - <Error> -- Failed to copy file [Errno 13] Permission denied: u'/var/lib/suricata/update/cache/92a59cfd53a431780ba3036effa26f9b-sslblacklist.rules'
```
This should be changed.

Reply:
Sorry, but I didn't get this. Do you mean that error message should have the filename where it is being copied?

Reply:
Yes. According to the string that is being written in the error message. i.e. `Failed to copy file ...` one would be expecting a file name following this line and not the error message.